app/views/ips/_table.html.erb

```diff
@@ -3,14 +3,19 @@
   <thead class="govuk-table__head">
     <tr class="govuk-table__row">
       <th class="govuk-table__header" scope="col">IP Address</th>
-      <th class="govuk-table__header" scope="col">Date Added</th>
+      <th class="govuk-table__header"></th>
     </tr>
   </thead>
   <tbody class="govuk-table__body">
     <% location.ips.each do |ip| %>
       <tr class="govuk-table__row">
-        <th class="govuk-table__header" scope="row"><%= ip.address %></th>
-        <td class="govuk-table__cell "><%= ip.created_at.strftime("%d %b %Y") %></td>
+        <td class="govuk-table__cell" scope="row"><%= ip.address %></td>
+        <% if ip.available? %>
+          <td class="govuk-table__cell">Available</td>
+        <% else %>
+          <td class="govuk-table__cell unavailable-ip">Activating tomorrow
```

Comment:
"Activating tomorrow" made me wonder what time tomorrow it will activate. Perhaps "Active tomorrow" or "Activating overnight" is more precise?

Reply:
Nice catch, I think this can be clearer - "Available from 6am tomorrow"? Doesn't tie us to some time in the night to restart that way.

Reply:
I can hear designers in my head saying "make it a negative to distinguish it!", so maybe _Not available until 6am tomorrow_.